src/services/compare-appian/serverless.yml

```diff
@@ -198,7 +209,7 @@ resources:
 stepFunctions:
   stateMachines:
     compareAlertFunc:
-      name: ${self:service}-${sls:stage}-compare-appian-alerting
+      name: ${self:service}-${sls:stage}-compare-alerting-appian
```

Comment:
can you do me a favor and change the compareAlertFunc name in compare-mmdl to match this pattern?

Reply:
Done!